.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,102 @@
+---
+name: Bug report
+description: Create a BUG report.
+title: "[BUG]: Title"
+labels: Bug
+body:
+- type: markdown
+  attributes:
+    value: |
+      Before you submit this request, please be aware that this is an
+      open source project, so you are invited to become a contributor to this project.
```

Comment:
"this is an open source project, so you are invited to become a contributor to this project."
Imo something like "contributions are welcome" would be enough for the second part of the sentence. 
Im not a fan of repeating "project".